sdk/breeze_connect/breeze_connect.py

```diff
@@ -1310,15 +1218,15 @@ def get_names(self, exchange_code, stock_code):
             compname = " ".join(requiredresult[' "CompanyName"'].to_string().split()[1:])
             if(" " in exchange):
                 exchange = " ".join(exchange.split()[1:])
-            #print(value,"val")
-            result = {}
-            result['exchange_code'] = exchange_code
-            result['exchange_stock_code'] = exchange
-            result['isec_stock_code'] = isec_stock
-            result['isec_token'] = token
-            result['company name'] = compname
-            #response = json.dumps(result,indent = 4)
+
+            result = {
+                    'exchange_code':exchange_code,
+                    'exchange_stock_code': exchange,
+                    'isec_stock_code':isec_stock,
+                    'isec_token': token,
+                    'company name':compname
+                }
     
-            return(result)
+            return result
         except Exception as e:
-            return({"Status" : "get_names() : data not found"})
+            self.error_exception(self.get_names.__name__,e)

```

Comment:
Good work removing all the unused variables, cleaning up redundant codes, and conde uniformity